packages/autotask-utils/src/types.ts

```diff
@@ -1,4 +1,4 @@
-import { SentinelConfirmation } from 'defender-sentinel-client/src/models/subscriber';
+export type SentinelConfirmation = number | 'safe' | 'finalized';
```

Comment:
I introduced the previous import. I prioritized not having duplicated code, so, I'm asking, do we have a plan to share models with the app somehow?

I'm worried it's getting messy, and the model discrepancies we have had been one of the main sources of reported issues.

Reply:
Yes, I'd want to set up a hook that pulls model definitions from the main repo into here. That should allow us to keep definitions centralized. But for now, I wanted to get the build to work.